### DISCUSSION TITLE
Setting up with MongoDB Atlas

### DISCUSSION THREAD

**Discussion by RZechlin** [hagopj13/node-express-boilerplate#51] [Q&A] [answered]
Hi,

First, thank you very much for this awesome boilerplate!
I'm very new to JS, Node, etc. and would like to connect the Cloud Database of MongoDB. 
I already had this working in another project with mongoose but here you're using Joi and have a more complicated setup.
When starting the application, obviously, an error that the MONGODB_URL is not defined gets thrown which lead me to the config.js. Is this the right spot to define the URL? 

And, is the following URL the right URL to connect to Mongo? It's what I used previously. 
`mongodb+srv://dbAdmin:<pw>@cluster0.zkahg.mongodb.net/<db>?retryWrites=true&w=majority"`

Thank you very much for taking the time to read my question!

Best,
Robert

**Comment by RZechlin** [comment, marked as answer]
As I said, I'm new and finally realized there's a .env file that handles this 😅

**Comment by hagopj13** [comment]
Hi @RZechlin, I'm glad you found the solution. I should probably add some information about this in the README.md.

**Reply by RZechlin** [reply]
The concept of having a file for the environmental variables makes a lot of sense for me after I understood it. As it was new to me, I didn't look for it though. Having it in the readme would me awesome.
